src/Compilers/CSharp/Test/Syntax/Parsing/LambdaParameterParsingTests.cs

```diff
@@ -1085,7 +1085,7 @@ public void TestNullCheckedNoParams()
         [Fact]
         public void TestNullCheckedDiscard()
         {
-            UsingDeclaration("Func<int> func1 = (_!) => 42;");
+            UsingDeclaration("Func<int, int> func1 = (_!) => 42;");
```

Comment:
Why did this change?

Reply:
Because when I copied the same test into the Semantic tests I got an error saying there were supposed to be 2 type params